src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days >= 7) {
+    sum = (days * 40) - 50;
+  } else if (days >= 3 && days < 7) {
+    sum = (days * 40) - 20;
+  } else {
+    sum = days * 40;
```

Comment:
No need to use if...else. It makes your code a bit complicated. 
Just return the full price immediately if the condition is true
![image](https://user-images.githubusercontent.com/96409563/231250423-08048199-8970-4180-b3f8-c434850f81c1.png)